dashboard/tests/test_dashboard.py

```diff
@@ -1012,5 +1013,20 @@ def test_dashboard_module_load(tmpdir):
     assert loaded_modules_actual == loaded_modules_expected
 
 
+def test_dashboard_module_no_warnings(enable_test_module):
+    # Disable log_once so we will get all warnings
+    from ray.util import debug
```

Comment:
@mattip can you tell me a bit more about what this test is for?

Reply:
gpustat had some problems with sub-dependencies, see [this comment](https://github.com/ray-project/ray/pull/26295#issuecomment-1189846832). This test makes sure the dashboard agent, which uses gpustat underneath, starts cleanly with no warnings or errors.